src/gui/editors/PianoRoll.cpp

```diff
@@ -2116,9 +2116,14 @@ void PianoRoll::mouseMoveEvent( QMouseEvent * me )
 				bool isUnderPosition = n->withinRange( ticks_start, ticks_end );
 				// Play note under the cursor
 				if ( isUnderPosition ) { testPlayNote( n ); }
-				// If note is the one under the cursor or is selected when alt is
-				// not pressed
-				if ( ( isUnderPosition && !isSelection() ) || ( n->selected() && !altPressed ) )
+				// If note is:
+				// Under the cursor, when there is no selection
+				// Selected, and alt is not pressed
+				// Under the cursor, selected, and alt is pressed
+				if ( ( isUnderPosition && !isSelection() ) || \
```

Comment:
Cosmetic but the [`\` before the newline](https://stackoverflow.com/a/14570370/3196753) is not needed since this line is not part of a preprocessor statement.  It's also inconsistent with the formatting of this file.